pkgs/applications/misc/k2pdfopt/default.nix

```diff
@@ -57,7 +57,28 @@ stdenv.mkDerivation rec {
       prePatch = ''
         cp ${src}/leptonica_mod/{allheaders.h,dewarp2.c,leptwin.c} src/
       '';
-      patches = [];
```

Comment:
Since we already patch the base `leptonica` in 19.03 (unlike master), wouldn't it be enough to just remove `patches = [];` so that it doesn't override it and inherit patches?

Reply:
Ah no I forgot to mention why we can't do that - it's because the CVE patches on vanilla leptonica are preceded by a build-system fix which doesn't apply on 1.74.4.

Reply:
Ah ok. Damn.